src/zappy_gui_src/Core/Entity/Entity.hpp

```diff
@@ -0,0 +1,125 @@
+/*
+** EPITECH PROJECT, 2022
+** ZAPPYMIRROR
+** File description:
+** Entity
+*/
+
+#ifndef ENTITY_HPP_
+#define ENTITY_HPP_
+
+#include <map>
+#include <string>
+#include <vector>
+
+/// \file src/zappy_gui_src/Core/Entity/Entity.hpp
+
+namespace gui::entity {
+enum player_orientation_e {
```

Comment:
Is the indentation good after calling namespace  ?

Reply:
need one indentation